packages/core/src/lib/Log.ts

```diff
@@ -14,10 +14,45 @@
  *  limitations under the License.
  */
 
+/**
+ * An interface for loggers.
+ */
+export interface Log {
+
+  verbose: boolean;
+
+  /**
+   * Prints a debug message to the Log. message is ignored if the Log is not set to verbose.
+   * @param message the message the be printed.
+   * @param args extra arguments for the console.
+   */
+  debug(message: string, ...args: string[]): void;
+
+  /**
+   * Prints an info message to the Log. message is ignored if the Log is not set to verbose.
+   * @param message the message the be printed.
+   * @param args extra arguments for the console.
+   */
+  info(message: string, ...args: string[]): void;
+
+  /**
+   * Prints an warning message to the Log. message is ignored if the Log is not set to verbose.
```

Comment:
Surely you only want "message is ignored if the Log is not set to verbose." on `debug` right? Not the other methods as well.

Reply:
I didn't right it, it is the same comment as Log had before.. should I change it?